pkg/docker/docker.go

```diff
@@ -544,7 +546,8 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
+// runContainerDockerRun funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
+// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
```

Comment:
there's a set of typos here, but let's discuss what the function is supposed to do first :)